package.json

```
@@ -30,7 +30,7 @@
         "@babel/env",
         {
           "targets": {
-            "node": "10"
+            "node": "16"
```

alternative solution for consideration: configure Babel for "current" Node.js version

```suggestion
            "node": "current"
```

seems to not break `npm test` or `yarn test-build` in my work area

---

On second thought, it would probably be safest to continue using a specific node version here ... considering this comment resolved now.